wolfssl/wolfcrypt/settings.h

```diff
@@ -1252,18 +1252,20 @@ extern void uITRON4_free(void *p) ;
 #endif
 
 #ifdef WOLFSSL_SGX
-    #define WOLFCRYPT_ONLY   /* limitation until IO resolved */
```

Comment:
Until the Windows version has been updated I think there should be a check here for what type of settings to use. Something like using _MSC_VER macro guard to retain the old settings on Windows.